README.md

```diff
@@ -12,13 +12,13 @@ The SPS frontend is a lightweight implementation of Epic Games' [Pixel Streaming
 ## Documentation
 
 ### Utilising the Scalable Pixel Streaming Frontend
-Below for a comprehensive guide to accessing the Scalable Pixel Streaming Frontend, including where to download the library and how to consume and customise it for your own projects.
+Below is a comprehensive guide to accessing the Scalable Pixel Streaming Frontend, including where to download the library and how to consume and customise it for your own projects.
 - [Scalable Pixel Streaming Frontend utilisation guide](./docs/frontend_utilisation_guide.md)
 
-### Migrating from Scalable Pixel Streaming frontend
+### Migrating from Scalable Pixel Streaming Frontend
 In general, the official Epic Games Pixel Streaming [Frontend docs](https://github.com/EpicGames/PixelStreamingInfrastructure/tree/master/Frontend) should cover most common usage cases (as our library is simply a thin wrapper on that). However, some SPS frontend specific docs are listed below:
```

Comment:
"However, some SPS ..." 

Change SPS to Scalable Pixel Streaming

Reply:
This typo has now been fixed